### ISSUE TITLE
Use `Ctrl + Alt + W` instead of `Ctrl + W` on Windows?

### ISSUE THREAD

**Issue by renkun-ken** [webBoxio/atom-hashrocket#8]
`CTRL + W`: Toggle watching current file's Hashrocket Markers

Sadly I'm a heavy user of `CTRL + W` to close tabs :( 

Is it possible to change the shortcut key to `Ctrl + Alt + W` instead?

**Comment by ZombieHippie** [comment]
I completely agree, just because `ctrl-w` is used in most major web browsers and text editors as the close tab command.

You should be able to just write it into your keymap file:

```cson
'.platform-win32 atom-workspace, .platform-linux atom-workspace':
  'ctrl-k ctrl-w': 'pane:close'
  'ctrl-w': 'core:close'
  'ctrl-alt-w': 'hashrocket:watchToggle'
```
